src/osd/osd_types.cc

```diff
@@ -890,6 +892,8 @@ boost::optional<uint64_t> pg_string_state(const std::string& state)
     type = PG_STATE_RECOVERY_UNFOUND;
   else if (state == "backfill_unfound")
     type = PG_STATE_BACKFILL_UNFOUND;
+  else if (state == "premerge")
```

Comment:
is this added to doc/dev/placement-group.rst later?

Reply:
added